api_factory/cli/generate.py

```diff
@@ -40,7 +42,12 @@ def generate(
 
     # Translate into a protobuf CodeGeneratorResponse;
     # if there are issues, error out appropriately.
```

Comment:
These comments seem more related to just the last line in this "paragraph" right?